src/content/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration.mdx

```diff
@@ -752,6 +752,259 @@ For information on ignored and expected errors, [see this page on Error Analytic
 
 </CollapserGroup>
 
+## AI monitoring [#ai-monitoring]
+
+This section includes Ruby agent configurations for setting up AI monitoring. 
+
+<Callout variant="important">
+You need to set [distributed_tracing](/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#dt-main) to `true` to capture trace and feedback data. It is turned on by default in Node.js agents 8.3.0 and higher. 
```

Comment:
Looks like the node info got copied here. 

The Ruby agent has distributed tracing enabled by default in 8.0.0 and higher.

Reply:
Fixed! Good catch